package.json

```diff
@@ -35,6 +35,7 @@
     "chai": "^4.1.0",
     "eslint": "^4.3.0",
     "eslint-config-shellscape": "^1.0.0",
+    "express": "^4.15.4",
```

Comment:
`express` is really too heavy for testing. I'd ask that [koa](http://koajs.com) be used instead. If you'd rather not reproduce these tests using koa I'd certainly understand and would be happy to do so after the PR is merged.